packages/botonic-plugin-nlu/src/index.js

```diff
@@ -2,16 +2,17 @@ import { NLU } from './nlu'
 
 export default class BotonicPluginNLU {
   constructor(options) {
+    let languages = Object.keys(options)
     return (async () => {
-      this.nlu = await new NLU(options)
+      this.nlu = await new NLU(languages)
       return this
     })()
   }
 
   async pre({ input, session, lastRoutePath }) {
     try {
-      let { intent, confidence, intents } = this.nlu.getIntents(input.data)[0]
-      Object.assign(input, { intent, confidence, intents })
+      let { intent, entities } = this.nlu.predict(input.data)
+      Object.assign(input, { ...intent, ...entities })
```

Comment:
shouldn't it be
`Object.assign(input, { intent, entities })
`?

```
>a={a:3}
{ a: 3 }

>Object.assign(a, {intent})
{ a: 3, intent: 'i' }
```

Reply:
fixed!